client/jetpack-onboarding/steps/contact-form.jsx

```diff
@@ -13,8 +14,13 @@ import DocumentHead from 'components/data/document-head';
 import FormattedHeader from 'components/formatted-header';
 import Tile from 'components/tile-grid/tile';
 import TileGrid from 'components/tile-grid';
+import { recordTracksEvent } from 'state/analytics/actions';
 
 class JetpackOnboardingContactFormStep extends React.PureComponent {
+	clickAddContactForm = () => {
+		this.props.recordTracksEvent( 'calypso_onboarding_contact_form_clicked' );
```

Comment:
I think we should synchronize on the prefix we'll be using for tracking those events: since we have several onboarding flows throughout Calypso, we'll need to prefix this one accordingly. 

Event name needs to start with `calypso_`,  and it should have a section prefix too, which in our case would be `jetpack_onboarding_`. However, since that results in a too huge prefix, we might go with `calypso_jpo_`.

Considering the above thoughts, this event could be altered to `calypso_jpo_contact_form_clicked`. 

What do you think?

Reply:
makes sense 👍